### PR TITLE
Add Repo name in Azure User Agent

### PR DESCRIPTION
**Pull request by vincent1173**

### CODE REVIEW

File: aks/entrypoint.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 set -e
-export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION}"
+export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION} REPOSITORY_${GITHUB_REPOSITORY}"
```
Comment (kmkumaran) [inline aks/entrypoint.sh]: Do you want to keep similar pattern of using _ as delimiter. The pattern will be `GITHUBACTIONS_<actionName>_<repoName>`

Comment (ds-ms) [reply aks/entrypoint.sh]: +1

Comment (vincentdass) [reply aks/entrypoint.sh]: updated. thanks

File: webapp/entrypoint.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 set -e
-export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION}"
+export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION}_${GITHUB_REPOSITORY}"
```
Comment (sachinma) [inline webapp/entrypoint.sh]: Do you need to consume this var in kudu npm as well?

Comment (vincentdass) [reply webapp/entrypoint.sh]: good suggestion. 
Will take it in V2 version.

Comment (kmkumaran) [reply webapp/entrypoint.sh]: sure. Make sense.

**Review by kmkumaran — APPROVED** [review]

**Review by thesattiraju — APPROVED** [review]

**Review by dikhakha — APPROVED** [review]

**Pull request merged**